### DISCUSSION TITLE
Section 3.1: formatting of "APE 0"

### DISCUSSION THREAD

**Discussion by bmerry** [astropy/astropy-v5.0-paper#72] [Text comments] [answered]
In some places it is written "APE 0" (with a space) and in other places "APE0" (no space).

**Comment by bmerry** [comment]
Whatever is chosen, "PEP 8" should probably get the same treatment.

**Comment by pllim** [comment, marked as answer]
Personally, I think it should be "APE 0" (with space).

**Reply by eerovaher** [reply]
Fixed by #88.
